lab13-jacob/routes/fantasy-routes.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+let Router = require('express').Router;
+let jsonParser = require('body-parser').json();
+
+let League = require('../model/fantasy');
+let router = module.exports = new Router();
+
+router.post('/leagues', jsonParser, (req, res, next) => {
+  if (Object.keys(req.body).length === 0 || !req.body.leagueName) {
```

Comment:
This condition is a bit redundant. In this case you only want to progress if the req.body.leagueName have a value, so no need to check for other key/vals.